backend/libs/response.ts

```diff
@@ -0,0 +1,44 @@
+import { APIGatewayProxyEvent, APIGatewayProxyResult } from 'aws-lambda';
+
+// @ts-ignore
+//eslint-disable-next-line @typescript-eslint/no-explicit-any
+type AttributeMap = { [key: string]: any };
+
+interface IRequestContext {
+    connectionId: string;
+    domainName: string;
+    stage: string;
+    authorizer: {};
+}
+
+interface IResponseBody {
+    status?: boolean;
+    error?: string;
+}
+
+type Body = IResponseBody | AttributeMap[] | AttributeMap;
+
+export interface IEvent extends Omit<APIGatewayProxyEvent, 'requestContext'> {
+    body: string;
+    methodArn: string;
+    requestContext: IRequestContext;
+    pathParameters: {
+        id: string;
+    };
+    source?: string;
+}
+
+function buildResponse(statusCode: number, body?: Body): APIGatewayProxyResult {
+    return {
+        statusCode,
+        body: JSON.stringify(body),
+    };
+}
+
+export const success = async (body?: Body) => {
+    return buildResponse(200, body);
+};
+
+export const failure = async (body?: Body) => {
+    return buildResponse(500, body);
```

Comment:
What happens if the API Gateway fails with a code other than 500 ?

Reply:
Nothing. As we're using websocket, the status code is not event that useful.

If you send a status code it is usually use as the response status code to the incoming request